Build fails with -Werror=maybe-uninitialized

In line 201 of host/utilities/bladeRF-cli/src/cmd/trigger.c variable is used before initialization. This makes build to fail under PyBOMBS, which runs compilation with -Werror=maybe-uninitialized

Steps to reproduce:
```
cd host
mkdir build
cd build
cmake -DCMAKE_CXX_FLAGS="-Werror=maybe-uninitialized" ..
make
```

Obtained result:
```
[...]
[ 76%] Building C object utilities/bladeRF-cli/CMakeFiles/bladeRF-cli.dir/src/cmd/version.c.o
[ 76%] Building C object utilities/bladeRF-cli/CMakeFiles/bladeRF-cli.dir/src/cmd/jump_boot.c.o
[ 76%] Building C object utilities/bladeRF-cli/CMakeFiles/bladeRF-cli.dir/src/cmd/mimo.c.o
[ 77%] Building C object utilities/bladeRF-cli/CMakeFiles/bladeRF-cli.dir/src/cmd/trigger.c.o
/home/jacek/build/bladeRF/host/utilities/bladeRF-cli/src/cmd/trigger.c: In function ‘cmd_trigger’:
/home/jacek/build/bladeRF/host/utilities/bladeRF-cli/src/cmd/trigger.c:201:12: error: ‘signal’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
     status = print_trigger(state, BLADERF_MODULE_TX, signal);
     ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/jacek/build/bladeRF/host/utilities/bladeRF-cli/src/cmd/trigger.c:186:28: note: ‘signal’ was declared here
     bladerf_trigger_signal signal;
                            ^~~~~~
cc1: all warnings being treated as errors
make[2]: *** [utilities/bladeRF-cli/CMakeFiles/bladeRF-cli.dir/build.make:496: utilities/bladeRF-cli/CMakeFiles/bladeRF-cli.dir/src/cmd/trigger.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1562: utilities/bladeRF-cli/CMakeFiles/bladeRF-cli.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

Expected result:
clean build

--- a/host/utilities/bladeRF-cli/src/cmd/trigger.c
+++ b/host/utilities/bladeRF-cli/src/cmd/trigger.c
@@ -191,6 +191,7 @@ static inline int print_all_triggers(struct cli_state *state)
         signal = BLADERF_TRIGGER_J51_1;
     } else {
         cli_err(state, "print_all_triggers", "Unknown board name");
+        return CLI_RET_INVPARAM;
     }
 
     status = print_trigger(state, BLADERF_MODULE_RX, signal);

